docs/CHANGELOG.asciidoc

```diff
@@ -50,6 +51,72 @@ Review important information about the {kib} 7.12.x releases.
+
+[float]
+[[enhancement-v7.12.1]]
+=== Enhancement
+Management::
+* Adds generated copy for all processors {kibana-pull}95507[#95507]
+
+[float]
+[[fixes-v7.12.1]]
+=== Bug Fixes
+Dashboard::
+* Fixes iFrame Filtering Issues {kibana-pull}95997[#95997]
+* Allows By Value Flow Without Visualize Save Permissions {kibana-pull}95951[#95951]
+* Fixes adding columns from doc viewer table tab {kibana-pull}95748[#95748]
+* Adds Discrete Library Option to Save Modal {kibana-pull}94589[#94589]
+Design::
+* Customize Panel Submit {kibana-pull}95704[#95704]
+Elastic Security::
+For the Elastic Security 7.12.1 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Machine Learning::
+* Data Frame Analytics creation: ensure job config validity persists when switching from/to form/editor {kibana-pull}94654[#94654]
+* Data Frame Analytics results: Ensure outlier detection results view displays feature influence correctly {kibana-pull}94493[#94493]
+* Fixes Index data visualizer not removing query string with loaded saved search {kibana-pull}94245[#94245]
+Management::
+* Transforms: Wizard displays warning callout for source preview when used with CCS against clusters below 7.10 {kibana-pull}96297[#96297]
+* Fixes updating deleting sessions from non-default space {kibana-pull}96123[#96123]
+* Disables navigation when a step is invalid {kibana-pull}95939[#95939]
+* Fixes 7.12 migration fail if the "timepicker:quickRanges" is null {kibana-pull}95767[#95767]
+* Fixes serialization and deserialization of user input for "patterns" fields {kibana-pull}94689[#94689]
+* Transforms: Fixes missing number of transform nodes and error reporting in stats bar {kibana-pull}93956[#93956]
+* Transforms: Improves error handling for transform wizard when Kibana index pattern or saved search fails to load {kibana-pull}93915[#93915]
+* Improves error message when localStorage quota is reached {kibana-pull}93779[#93779]
+Maps::
+* Fixes Kibana does not recognize a valid geo_shape index when attempting to create a Tracking Containment alert {kibana-pull}96633[#96633]
+* Shows empty list when all saved maps in list deleted {kibana-pull}95126[#95126]
+* Fixes tooltips with timestamp appear as epoch {kibana-pull}95106[#95106]
+Monitoring::
+* Fixes issue with loading logstash node page under standalone cluster {kibana-pull}93617[#93617]
+* Fetches status once and change fetchStatus to support an array of clusters {kibana-pull}91749[#91749]
+Operations::
+* Improves performance of data stream API {kibana-pull}97058[#97058]
+* Migrations v2 ignore fleet agent events {kibana-pull}96690[#96690]
+* Data Frame Analytics: Fix scatterplot matrix boilerplate visibility with no fields selected {kibana-pull}96590[#96590]
+* Data Frame Analytics: Don't allow user to pick an index pattern or saved search based on CCS {kibana-pull}96555[#96555]
```

Comment:
These two items are also ML-related ones. If there is a reason why these are under the Operations section, please ignore my comment.

Reply:
They are definitely ML related. I'll move them.